test/parallel/test-http-server-unconsume.js

```diff
@@ -14,6 +14,8 @@ const server = http.createServer(function(req, res) {
     received += data;
   });
 
+  assert.strictEqual(req.socket.on, req.socket.addListener);
```

Comment:
Should a test be added to check the value of `req.socket.prependListener`?

Reply:
> Should a test be added to check the value of `req.socket.prependListener`?

To be clear: That's not a rhetorical question. I don't know if it should be added or not.

Reply:
It should.
Comparing `req.socket.prependListener` with `Net.prototype.prependListener` to avoiding some to change its behavior